scripts/approvalYears.py

```diff
@@ -923,10 +926,10 @@ def writeCBERBLAs(fpout, uniiPT, uniiALL):
                         early[akey][-5] = entry[-5]
 
 
-        for key in early.keys():
+        for key in list(early.keys()):
```

Comment:
Same as above.

Reply:
Python3 versus python2 issue. 2to3 automagically has made this change as .keys() no longer returns a list in python3